src/lib/type/index.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+export type Triggers= {
+  setName: Function,
+  setEmail: Function,
+  setCompany: Function,
+  setPhone: Function,
+}
+
+export type InputFieldParams = {
+    captureField: Function,
+    name: string,
+    triggers: Triggers
+}
+
+export type Values = {
+  name: string,
+  email: string,
+  company: string,
+  phone: string
+}
+
```

Comment:
PreventDefaultFunction and eventTargetIdValue are the same object type, but different use. For this application it works because of it size, but as mentioned before, lets keep in mind scalability. If you combine this into one typed object you can use the one type for both locations. 
Later one down the road when another developer comes in to check to see how they can repurpose what is already there, it is available. As two individual types, they are more likely to be skipped over and add a new type, or left alone to a VERY specific use case.  
So as you start to work in a multi-facet application; yes, you need to consider keeping things for their use case, BUT, you also need to always, ALWAYS, consider scalability. Yes, think of what you need it for, however, also consider a few other ways it can be used. You do not need to code for its other use cases when thinking about it, just code for your use case; yet, leave it open for the ability for another developer to reused it for their needs.